src/releaseAndTag.js

```diff
@@ -107,7 +106,7 @@ const createTempDirectory = async (name) => {
   if (exists) {
```

Comment:
Not directly related to this change, but this bit of existing code is a tiny race condition. Not really a big deal given the context, but we should talk about what the desired behavior is in certain cases and eliminate usage of `fs.exists()`.